quarto/table-templates/template-table_32.qmd

```diff
@@ -40,6 +40,64 @@ Required variables:
+| `show_colcounts` | (`flag`) Whether column counts should be printed.                                                                                                                                                                                                      | `TRUE`       |
+| `arm_var`        | (`character`) Arm variable used to split table into columns.                                                                                                                                                                                           | `"ARM"`      |
+| `lbl_overall`    | (`character`) If specified, an overall column will be added to the table with the given value as the column label.                                                                                                                                     | `NULL`       |
+| `prune_0`        | (`flag`) Whether all-zero rows should be removed from the table.                                                                                                                                                                                       | `FALSE`      |
+| `annotations`    | (named `list` of `character`) List of annotations to add to the table. Valid annotation types are `title`, `subtitles`, `main_footer`, and `prov_footer.` Each name-value pair should use the annotation type as name and the desired string as value. | `NULL`       |
+
+Source code for this function is available [here](https://github.com/pharmaverse/falcon/blob/main/R/fda-table_32.R).
+
+## gt Table
+
+```{r tbl_gt, message=FALSE, warning=FALSE}
+# Load Libraries & Data
+library(scda)
+library(falcon)
+library(dplyr)
+library(gt)
+library(gtsummary)
+library(tern)
+
+adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl")
+advs <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "advs")
+
+# Output table
+tbl <- make_table_32_gtsum(advs = advs, alt_counts_df = adsl)
+tbl
+```
+
+## gt Table Setup
+
+```{r tbl_gt, eval=FALSE, echo=TRUE}
+```
+
+## Function Details: `make_table_32_gtsum`
+
+### `make_table_32_gt()`
+
+------------------------------------------------------------------------
+
+Required variables:
+
+-   **`advs`**: `SAFFL`, `USUBJID`, `AVISITN`, `PARAMCD`, `AVAL`, `AVALU`, and the variables specified by `arm_var`.
+-   **`alt_counts_df`** (if specified): `SAFFL`, `USUBJID`, and the variable specified by `arm_var`.
+
+| Argument        | Description                                                                                                        | Default              |
+|:-----------------|:----------------------------------|:-----------------|
+| `advs`          | (`data.frame`) Dataset (typically ADVS) required to build table.                                                   | *No default*         |
```

Comment:
adsl included here too

Reply:
The current version of template-table_32.qmd includes alt_counts_df on lines 48 and 95, the same way as table 6 or others. Should it be different here?

I'm gonna modify the branch also to include saffl_var to the gt table.

Reply:
But the `gtsum` function [currently requires](https://github.com/pharmaverse/falcon/blob/33d76e8b737c8de601fd8c016e20a6017121cbc7/R/fda-table_32.R#L124-L130) the adsl data (apart from the alt_counts_df). Should the function be reworked then to remove the adsl processing?

Reply:
I misunderstood that point, sorry. I'm gonna find a way to add adsl parameter to the documentation, or think of a way to remove the adsl processing (if possible - the issue is on the population count for percentages).